src/algebra/lie/basic.lean

```diff
@@ -1743,6 +1969,45 @@ begin
   { intros I J hI hJ, apply lie_algebra.is_solvable_add R L; [exact hI, exact hJ], },
 end
 
+/-- The `→` direction of this lemma is actually true without the `is_noetherian` assumption. -/
+lemma lie_ideal.solvable_iff_le_radical [is_noetherian R L] (I : lie_ideal R L) :
+  is_solvable R I ↔ I ≤ radical R L :=
+begin
+  split; intros h,
+  { exact le_Sup h, },
+  { apply le_solvable_ideal_solvable h, apply_instance, },
+end
+
+/-- A semisimple Lie algebra is one with trivial radical. -/
+class is_semisimple : Prop :=
+(semisimple : radical R L = ⊥)
```

Comment:
It's worth noting that no less a person than [Jim Humphreys claimed](https://mathoverflow.net/questions/149391/on-radicals-of-a-lie-algebra#comment383669_149391) there is no agreement on standard meaning of semisimple outside of characteristic zero.

Nevertheless I am a least consistent with Seligman who literally wrote the book on modular Lie algebras. He says:
![image](https://user-images.githubusercontent.com/7734364/106145617-aacfb280-616d-11eb-89a2-c63a7dfd2b5e.png)

Reply:
Feel free to write a short remark about this in the docstring. I'm sure I will forget otherwise.

Reply:
Done!